### PR TITLE
LBSD 2515 loadmoreposts button disappears

### PR DESCRIPTION
**Pull request by dev-tsoni**

### CODE REVIEW

File: server/liveblog/themes/themes_assets/default/js/theme/handlers.js [hunk 1]
```diff
@@ -84,15 +83,15 @@ var buttons = {
...
     },
 
     "[data-js-orderby_descending]": () => {
-      loadSort('descending');
+      if(isOrderChanged('descending')) loadSort('descending');
     },
 
     "[data-js-orderby_editorial]": () => {
-      loadSort('editorial');
+      if(isOrderChanged('editorial')) loadSort('editorial');
```
Comment (eos87) [inline server/liveblog/themes/themes_assets/default/js/theme/handlers.js:94]: I'm not sure about this. The conditions don't make sense. Would it be better to have the condition checked inside `loadSort` method?

File: server/liveblog/themes/themes_assets/angular/theme.json [hunk 2]
```diff
@@ -89,8 +89,8 @@
     ],
     "scripts": [
         "https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js",
-        "https://ajax.googleapis.com/ajax/libs/angularjs/1.5.11/angular.min.js",
-        "https://cdnjs.cloudflare.com/ajax/libs/angular-resource/1.5.11/angular-resource.min.js",
+        "https://ajax.googleapis.com/ajax/libs/angularjs/1.5.11/angular.js",
+        "https://cdnjs.cloudflare.com/ajax/libs/angular-resource/1.5.11/angular-resource.js",
```
Comment (eos87) [inline server/liveblog/themes/themes_assets/angular/theme.json:93]: Please move this back to `.min.js` versions as `scripts` is for production.

File: server/liveblog/themes/themes_assets/default/js/theme/handlers.js (hunk 1 above)
Comment (dev-tsoni) [reply server/liveblog/themes/themes_assets/default/js/theme/handlers.js:94]: Yes, will move it to loadSort.

File: server/liveblog/themes/themes_assets/angular/theme.json (hunk 2 above)
Comment (dev-tsoni) [reply server/liveblog/themes/themes_assets/angular/theme.json]: done

**Pull request merged**